website/sales/models/shift.py

```diff
@@ -0,0 +1,189 @@
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.db.models import (
+    Sum,
+    Q,
+    Count,
+)
+from django.db.models.expressions import RawSQL
+from django.utils import timezone
+from django.utils.translation import gettext_lazy as _
+from queryable_properties.managers import QueryablePropertiesManager
+from queryable_properties.properties import (
+    RangeCheckProperty,
+    queryable_property,
+)
+
+from activemembers.models import MemberGroup
+from sales.models.product import ProductList
+
+
+class Shift(models.Model):
+    class Meta:
+        permissions = [
+            ("bypass_manager", _("Can bypass shift manager restrictions")),
```

Comment:
```suggestion
            ("override_manager", _("Can access all shifts as manager")),
```
I'd like to see this called an override instead of a bypass, since that is the name the permission also has in the events app.